app/src/main/java/com/example/ohthmhyh/CustomAdapterHF.java

```diff
@@ -20,7 +20,7 @@
 /**
```

Comment:
Why are changes made in this file?

Reply:
This is android studio being weird and recognised that I changed the file. If you looked at the changes, they are meaningless.